scripts/send-slack-message-pr.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+SLACK_BOT_TOKEN=$1
+PULL_REQUEST_NUMBER=$2
```

Comment:
what if this doesn't exist?

in the current state you run this for branch builds as well from what I can tell